<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoutubeDowloader</class>
 <widget class="QMainWindow" name="YoutubeDowloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans Condensed</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloader</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="urlText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="directoryText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="titleSoftware">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Condensed</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Youtube Downloader</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Descargar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qualitySelect">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Media</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baja</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="qualityLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Calidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="directoryLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Directorio</string>
    </property>
   </widget>
   <widget class="QLabel" name="urlLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="directorySelect">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="formatSelect">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mp4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mp3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>webm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>aac</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>m4a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ogg</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="formatLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Formato</string>
    </property>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>382</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>directorySelect</sender>
   <signal>clicked()</signal>
   <receiver>YoutubeDowloader</receiver>
   <slot>chooseDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>YoutubeDowloader</receiver>
   <slot>downloadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>downloadFile()</slot>
  <slot>chooseDirectory()</slot>
 </slots>
</ui>
